README.md

```diff
@@ -1,2 +1,58 @@
-# static-code-analysis-tool
-Tool for performing static code analysis for Ballerina projects.
+# Ballerina Static Code Analysis Tool
+
+## Overview
+Static Code Analysis (SCA) is an approach that uses tools to examine code without execution. 
+It's used for identifying potential issues like bugs, vulnerabilities and style violations.
+SCA improves software quality by early issue detection, creating better maintainability, and
+providing enhanced security. The Ballerina scan tool provides a set of command line tools to
+statically analyze Ballerina files and report analysis issues.
+
+## Prerequisites
+1. openJDK 17 ([Adopt OpenJDK](https://adoptium.net/temurin/releases/?version=17) or any other OpenJDK distribution)
+
+2. [Ballerina version: 2201.8.5](https://ballerina.io/downloads/)

```

Comment:
Shouldn't be tied to a b7a version since this is the repo for active development.